crates/services/upgradable-executor/src/lib.rs

```diff
@@ -0,0 +1,38 @@
+#![deny(clippy::arithmetic_side_effects)]
+#![deny(clippy::cast_possible_truncation)]
+#![deny(unused_crate_dependencies)]
+#![deny(warnings)]
+
+#[cfg(feature = "wasm-executor")]
+use wasmtime::{
+    Config,
+    Engine,
+    Module,
+};
+
+pub mod config;
+pub mod executor;
+
+#[cfg(feature = "wasm-executor")]
+pub mod instance;
+
+/// The WASM bytecode of the native executor.
```

Comment:
This comment is confusing. "of the native executor"? Do you just mean the "executor"? It's not native anymore if it's Wasm.